modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java

```diff
@@ -427,29 +447,29 @@ private static Type getType(String comp0, String ssl0) {
             String comp = String.valueOf(comp0);
             boolean ssl = Boolean.valueOf(ssl0);
 
-            if (comp.equals("NO_COMPRESSION") && ssl)
+            if ("NO_COMPRESSION".equals(comp) && ssl)
                 return Ssl;
-            if (comp.equals("NO_COMPRESSION") && !ssl)
+            if ("NO_COMPRESSION".equals(comp) && !ssl)
                 return No;
-            if (comp.equals("LZ4") && !ssl)
+            if ("LZ4".equals(comp) && !ssl)
                 return LZ4;
-            if (comp.equals("LZ4") && ssl)
+            if ("LZ4".equals(comp) && ssl)
```

Comment:
Maybe to use static final strings for types?

Reply:
Done